ibm/mas_devops/roles/cp4d_db2wh_manage_hack/tasks/main.yml

```diff
@@ -26,10 +26,12 @@
     name: "{{ cp4d_db_config_configmap_name }}-{{ db2wh_instance_name }}"
     namespace: "{{ cpd_meta_namespace }}"
   register: cp4d_db_config_info
+  when: db2wh_load_from_config == true
```

Comment:
`db2wh_load_from_config` was eliminated in master to streamline things ... can we do the same here, there's no need for a load from config == false mode as there is only one way to provision now (cp4d_db2wh_api replaced cp4d_db2wh instead of being an alternative role)

Reply:
Let me test it out some more and see if I can get it removed, the reason why I added it in first place was because db2wh in v4 is installed via db2ucluster currently and it does not require a config-map to store the information for it, and it was failing on the manage-hack role as I don't have a configmap for this version.
So added this check to just use the configmap for db2wh v3.5

Reply:
remove the need of `db2wh_load_from_config` and now if `cpd_version == 'cpd40'` then we don't load the config map, if `cpd_version == 'cpd35'` then we load the config map as rest api expects.